tools/check_format.py

```diff
@@ -370,6 +377,9 @@ def checkSourceLine(line, file_path, reportError):
     if invalid_construct in line:
       reportError("incorrect protobuf type reference %s; "
                   "should be %s" % (invalid_construct, valid_construct))
+  for invalid_construct, valid_construct in CPP_STD_ERRORS.items():
+    if invalid_construct in line:
+      reportError("cpp std: %s should be replaced by %s" % (invalid_construct, valid_construct))
```

Comment:
Can we phrase this:

 function %s should be replaced with standard library function %s

and give the entire elaborated version of that as the expected error message in check_format_test_helper.py.

Reply:
Fixed except for word "function". I believe there will be type as well.